src/macaron/config/global_config.py

```diff
@@ -60,9 +64,18 @@ def load(
         self.debug_level = debug_level
         self.local_repos_path = local_repos_path
         self.gh_token = gh_token
-        self.policy_path = policy_path
         self.resources_path = resources_path
 
+        # Find the policies
+        policy_files = []
+        for policy_path in policy_paths:
+            for policy_file_path in glob.iglob(f"{policy_path}/**", recursive=False):
```

Comment:
For number 1 I don't think it is True as per the documentation [here](https://docs.python.org/3/library/glob.html#glob.glob), `iglob` is an Iterator version of `glob.glob` so I don't think the behavior is different.

I tried the following snippet in my Python 3.11 virtual environment:
```
import glob

list(glob.iglob("./test_dir/**", recursive=False)) # will only print out the first level of test_dir (including files and directories)
```

Reply:
1. It recurses 1 level which is the desired behaviour; if a folder is specified then it yeilds the folder and all the files in the folder, and if a file is specified it just yeilds that file.
2. Its not necessarily a YAML file it could also be a cue file. The filetype is checked when the policy is constructed so no need to do it twice.

Reply:
> 1. if a folder is specified then it yeilds the folder and all the files in the folder, and if a file is specified it just yeilds that file.

 The `f"{policy_path}/**"` pattern won't match a single file though, e.g., if `policy_path` is `policy.cue`.

Reply:
Ah of course, thanks. I it back to just explicitly checking if its a folder or file in this commit  https://github.com/oracle-samples/macaron/pull/46/commits/766a2bf3d6b1d52cd639d68e59d4013e835c655f